internal/restinterface/externalhandler/externalhandler.go

```diff
@@ -351,6 +359,101 @@ SEND_RESP:
 	h.helper.Response(w, respEncryptBytes, http.StatusOK)
 }
 
+// APIV1RequestCloudSyncmgrPost handles cloudsync publish request from service application
+func (h *Handler) APIV1RequestCloudSyncmgrPost(w http.ResponseWriter, r *http.Request) {
```

Comment:
If I call this 10 times with the same host, will 10 mqtt client go routines be created?

Reply:
currently it is not storing the client data. Also one more thing when same host is sending the conenctn request the previous connection is closed and new is opened.

Reply:
Wanted to make the base code with single connection with the client. Going forward we think of scenarios where multiple service applications can call the send API to send data to different broker URL's. So in that case maintaining separate mqtt client instance for each service application would be better(and only one instance for each host). Use of db to maintain the same or so could be option. We plan to do this once the base code is merged as part of next steps.